ui/src/cloud/actions/orgsettings.ts

```diff
@@ -0,0 +1,48 @@
+// API
+import {getOrgSettings as getOrgSettingsAJAX} from 'src/cloud/apis/orgsettings'
+
+// Constants
+import {FREE_ORG_HIDE_UPGRADE_SETTING} from 'src/cloud/constants'
+
+// Types
+import {GetState, OrgSetting} from 'src/types'
+
+// Selectors
+import {getOrg} from 'src/organizations/selectors'
+
+export enum ActionTypes {
+  SetOrgSettings = 'SET_ORG_SETTINGS',
+}
+
+export type Actions = SetOrgSettings
+
+export interface SetOrgSettings {
+  type: ActionTypes.SetOrgSettings
+  payload: {orgSettings: OrgSetting[]}
+}
+
+export const setOrgSettings = (settings: OrgSetting[] = []): SetOrgSettings => {
```

Comment:
in an attempt to have actions be a little less verbose, we're trying to remove the `ActionTypes + Actions + interface + action` paradigm and replace it with the `ReturnType<typeof> + as const` pattern you can find in the resource normalization work:
https://github.com/influxdata/influxdb/blob/master/ui/src/dashboards/actions/creators.ts#L29

Reply:
Good catch, thanks. But we are just doing the `ReturnType<typeof>` part only. The normalization is a bit out of scope, because _all_ cloud resources should be normalized together and that is a bigger and separate effort.